lib/Server.js

```diff
@@ -553,12 +551,15 @@ class Server {
 
     defaultFeatures.push('magicHtml');
 
-    if (
-      contentBase !== false &&
-      (options.serveIndex || options.serveIndex === undefined)
-    ) {
+    // checking if it's set to true or not set (Default : undefined => true)
+    options.serveIndex = options.serveIndex || options.serveIndex === undefined;
```

Comment:
Better avoid modify `options` because code below also can use options and modification original options is not good idea better keep this in separate variable, it is good practice.
I think we can rewrite this to:
```
const serveIndexValue = options.serveIndex ? options.serveIndex : true;
```

If it is `null/undefined` we use `true` otherwise we get original value.

Reply:
@evilebottnawi thank you for the advice, I set the option value to a separate variable `this.serveIndex`